docs/testing-contracts.rst

```diff
@@ -5,13 +5,13 @@ Testing a Contract
 
 For testing Vyper contracts we recommend the use of `pytest <https://docs.pytest.org/en/latest/contents.html>`_ along with one of the following packages:
 
+    * `Titanoboa <https://github.com/vyperlang/titanoboa>`_: A Vyper interpreter, pretty tracebacks, forking, debugging and deployment features. Maintained by the Vyper team.
     * `Brownie <https://github.com/iamdefinitelyahuman/brownie>`_: A development and testing framework for smart contracts targeting the Ethereum Virtual Machine
```

Comment:
Replace brownie with ape?

Reply:
brownie is still alive, we could add an additional entry for ape (in a different PR though)

Reply:
Alive and Ben wanting to attract more users to maintain it for are two different things

Reply:
i mean last release was 3 days ago https://github.com/eth-brownie/brownie/releases. but also it just seems out of scope to make changes here